package.json

```diff
@@ -39,7 +39,8 @@
     "pre-commit": "npm run autofix && npm test",
     "autofix": "eslint . --fix",
     "test-spec": "tap test/*spec.js",
-    "test-perf": "node test/performance.js"
+    "test-perf": "node test/performance.js",
+    "start": "npm install && npm start",
```

Comment:
This looks like it will call itself in an endless loop.

Reply:
dont update package.json
but please update content-types.js

Reply:
You should undo this change then; pull-requests are all-or-nothing, I can't merge in part while leaving out another part. (You can make different branches with different sets of changes, and then a pull request would only get the changes from one branch.)